src/public/secure/admin.ejs

```diff
@@ -104,6 +104,22 @@
             </div>
         </div>
     </div>
+
+    <div class="modal fade" id="fileDetailsModel" tabindex="-1" aria-labelledby="fileDetailsModelLabel"
+         aria-hidden="true">
+        <div class="modal-dialog" style="max-width:50% !important;">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h1 class="modal-title fs-5" id="fileDetailsModelLabel">File Details</h1>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <div class="modal-body" id="fileDetailsModelInfoContent">
+                </div>
+                <div class="modal-footer">
```

Comment:
you forgot the close button

Reply:
Was on purpose - close is X at top - no ?

Reply:
Changed per request to include close button